### ISSUE TITLE
Editor toolbar buttons overflow

### ISSUE THREAD

**Issue by subjective** [peer/doc#163]
I'm using the merge/fork test version for Unifesp.  See below:

![peerdoc-bug-1](https://user-images.githubusercontent.com/5214824/48739307-9274bc80-ec21-11e8-9d95-2588e948d712.png)
![peerdoc-bug-2](https://user-images.githubusercontent.com/5214824/48739308-94d71680-ec21-11e8-84dc-ed0276894532.png)

**Comment by mitar** [comment]
Yes, they do, these are visible only on testing site for debugging.

**Comment by mitar** [comment]
But feel free to suggest solutions here. :-)
